gobblin-core/src/main/java/gobblin/writer/AvroHdfsDataWriter.java

```diff
@@ -11,8 +11,14 @@
 
 package gobblin.writer;
 
+import com.google.common.base.Preconditions;
```

Comment:
Good question. :), The answer is it's not clearly defined, but I would suggest putting them right before the `gobblin` ones.